src/bp/core/services/cms.ts

```diff
@@ -436,7 +451,32 @@ export class CMSService implements IDisposeOnExit {
     }, {})
   }
 
-  getTranslatedElement(formData: object, contentType: ContentType, lang: string, defaultLang?: string) {
+  async translateContentProps(botId, fromLang, toLang) {
+    const elements = await this.listContentElements(botId, undefined, { from: 0, count: UNLIMITED_ELEMENTS })
+    for (const el of elements) {
+      if (!fromLang) {
+        // Translating a bot content from the original props
```

Comment:
do we want to mark this as deprecated ?

Reply:
I think we should keep it, there could be some use case in the future